sdcm/utils/common.py

```diff
@@ -182,6 +182,16 @@ def generate_random_string(length):
         random.choice(string.ascii_uppercase + string.digits) for x in range(length - 1))
 
 
+def str_to_bool(bool_as_str: str) -> bool:
```

Comment:
could reuse: `from distutils.util import strtobool`

Reply:
It is deprecated: https://peps.python.org/pep-0632/#migration-advice

Reply:
Following this change, we should replace other usages as well, we have one in sct_config.py